packages/analytics/amplify_analytics_pinpoint/analysis_options.yaml

```
@@ -1 +1,5 @@
 include: package:amplify_lints/library.yaml
+
+analyzer:
+  exclude:
+    - "**/**/**/*.g.dart"
```

`**/**/**/` should be reducible to `**/`
```suggestion
    - "**/*.g.dart"
```

---

Merging with next was unsuccessful. Please fix and address Jordan's comment:

```yaml
analyzer: 
  errors:
    implementation_imports: error #TODO(equartey): Remove when lint is enforced project-wide    
  exclude:
    - "**/*.g.dart"
```